doc/authoring_help.md

```diff
@@ -147,10 +147,11 @@ The first example is only supported on the `latest` and `2018-03-01-hybrid` prof
           supported-profiles: 2017-03-09-profile
 ```
 
-Here is how this looks in CLI `--help`:
+Here is how this looks in Azure CLI `--help`:
 
 On profiles `latest` and `2018-03-01-hybrid`.
-```
+
+```console
```

Comment:
I don't think this is necessary as GitHub will not render it with any additional highlighting:

![image](https://user-images.githubusercontent.com/4003950/104869159-45ffa580-5980-11eb-905e-584d3ddd5fc2.png)

Reply:
Hi @jiasli 

As I understand, this is based on best practices. Code blocks should not be left blank, and "console" is used as a generic tag. In this case, it might not be correct anyway. I wasn't sure if this block is considered Azure CLI or not, since it has an 'az' reference. @dbradish-microsoft, can you verify?